server/app/dal/CassandraCluster.scala

```diff
@@ -44,14 +42,14 @@ class CassandraConfig @Inject()(configuration: Configuration, environment: Envir
     val hostsConf = "devgym.db.cassandra.hosts"
     val dockerConf = "devgym.db.cassandra.docker"
 
-    def hosts: Seq[String] = {
+    def getHosts: Seq[String] = {
```

Comment:
yeah, there were words "hosts", kind of kalambur and hard to read. If you want you can change name back.